bin/all_sky_search/pycbc_fit_sngls_over_multiparam

```diff
@@ -138,6 +138,35 @@ def smooth(nabove, invalphan, ntotal, dists, smoothing_method, **kwargs):
     return _smooth_dist_func[smoothing_method](nabove, invalphan,
                                                ntotal, dists, **kwargs)
 
+# This is used to pre-calculate the templates which could be cut
+# before calculating distances
+# The value is used to determine the 1D-extent of the used templates;
+# number of smoothing lengths to cut anything outwith this range.
```

Comment:
this is a bit repetitive .. suggest 'Number of smoothing lengths around the current template where distances will be calculated'